.editorconfig

```diff
@@ -260,3 +260,16 @@ resharper_csharp_wrap_list_pattern = chop_if_long
 
 [src/Aquifer.Data/Migrations/*.cs]
 generated_code = true
+
+[*.{cs,vb}]
+dotnet_style_coalesce_expression = true:error
+dotnet_style_null_propagation = true:error
+dotnet_style_prefer_is_null_check_over_reference_equality_method = true:suggestion
+dotnet_style_prefer_auto_properties = true:silent
+dotnet_style_object_initializer = true:suggestion
+dotnet_style_collection_initializer = true:silent
+dotnet_style_operator_placement_when_wrapping = beginning_of_line
+tab_width = 4
+indent_size = 4
+end_of_line = crlf
```

Comment:
Do we use crlf or lf?  I think this might have just pulled my local settings and populated the `.editorconfig` file.

Reply:
Mine have this line in it